news/2019-11-11-winter-2019-fanart-contest.md

```diff
@@ -0,0 +1,40 @@
+* **Attribution (artist names, Twitter handles and other things like pixiv links, socials, etc) can be included in your entry, but must be situated either on the bottom left, or bottom right of your entry, and must not be overly large.** We're not going to judge this one too hard in terms of size, but please don't make them massive. People have been including attribution links in their entries for a while now and we figured we may as well formalize them loosely at this point, just make sure they're not like, gigantic. Anything taller than 300 pixels in height is probably too big. We want your art to shine foremost, and this kind of stuff *does* influence the voting a little more than we'd like, but we can't exactly ask you to not take credit for your own work, could we?
+* **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.** Anything inside the dotted circle will NOT be visible during voting phase, and will be heavily obscured by the main menu cookie if it makes it to the game client.
+* **DO NOT LEAVE THE DOTTED osu! COOKIE TEMPLATE VISIBLE IN YOUR SUBMISSION!** Please don't do this. It doesn't need to be there when you're done drawing, it just makes your work look worse.
+* **ACCOUNT FOR THE DOTTED osu! COOKIE TEMPLATE IN THE COMPOSITION OF YOUR WORK!** We've already said this, but anything inside the dotted circle will *not* be seen during voting phase and will be very difficult to see on the main menu. You still need to draw something there — don't just leave it blank, but be creative about it. Check out previous contest entries for inspiration!
+* **MAKE SURE YOU DON'T HAVE THE DOTTED osu! COOKIE TEMPLATE VISIBLE IN YOUR WORK!** What's this, the same rule repeated three times in three slightly different ways? What could this POSSIBLY MEAN? If you're not following, it means to TRIPLE CHECK that you've not left the template visible in your work. 
+* **Entries MUST meet the resolution requirements for a main-menu piece - namely 2732x1536. THIS IS CHECKED BY A SCRIPT AND ENTRIES THAT DO NOT MATCH THIS REQUIREMENT WILL BE DISCARDED AUTOMATICALLY.** Just in case you missed it, a script automatically discards entries that don't meet the resolution requirements, even if they're off by only a few pixels. This is the #1 reason why people don't see their artwork in the end voting, and many excellent entries have met this tragic, tragic fate. Export your entry at the proper size!
```

Comment:
```suggestion
* **Entries MUST meet the resolution requirements for a main-menu piece — namely 2732x1536. THIS IS CHECKED BY A SCRIPT AND ENTRIES THAT DO NOT MATCH THIS REQUIREMENT WILL BE DISCARDED AUTOMATICALLY.** Just in case you missed it, a script automatically discards entries that don't meet the resolution requirements, even if they're off by only a few pixels. This is the #1 reason why people don't see their artwork in the end voting, and many excellent entries have met this tragic, tragic fate. Export your entry at the proper size!
```